tests/cypress/integration/reportRepair/repairDescription.spec.js

```diff
@@ -43,45 +43,6 @@ describe('repair description', () => {
     });
   });
 
-  context('When a user uploads an image with the wrong format', ()=>{
```

Comment:
These tests are still relevant whilst MVP is being supported.
Could a separate CI pipeline step be created for run the MVP only tests?